packages/react-ui/src/widgets/ComparativeFormulaWidgetUI.js

```diff
@@ -66,7 +67,7 @@ function ComparativeFormulaWidgetUI({
   }
 
   return (
-    <div>
+    <div className={classes.formulaChart}>
```

Comment:
I've removed this class in the previous PR, as it has no content

Reply:
oh sorry, I think it was there for something

Reply:
done too